rickynyairo/GitSla
# #164523024: send user all unread notifications

#### What does this PR do?
- creates slack message models
- creates slack actions router
- adds a button to view unread notifications
- sends a user all their unread notifications

#### Description of Task to be completed
- The bot should be able to send all a user's unread notifications.

#### How should this be manually tested?
- use `git fetch` and switch to the branch `ft-github-notifications-164523024`
- use the command `/gitbot` and click the button `Check Unread Notifications`
-  a list of notifications should show

#### Any background context you want to provide?
- authentication is not yet set up, so all notifications shown are from my user `njeri-ngigi`
- the list is not paginated yet
- if no notifications are available, a message will show you just that message.


#### What are the relevant pivotal tracker stories?
- [#164523024 send user all unread notifications](https://www.pivotaltracker.com/story/show/164523024)
- [#164522984 create slack message models](https://www.pivotaltracker.com/story/show/164522984)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/28973383/56193372-257f1a80-6039-11e9-8a2a-6f28aafcee2e.png)
![image](https://user-images.githubusercontent.com/28973383/56193378-287a0b00-6039-11e9-87d2-b61d97430081.png)

#### Checklist
